include/ap.h

```diff
@@ -0,0 +1,28 @@
+#ifndef _AP_H_
+#define _AP_H_
+
+#include <WiFi.h>
+#include <dataqueue.h>
+#include <esp_wifi.h>
+#include <layer2.h>
+#include "message.h"
+#include "wifinode.h"
+
+class AccessPoint : public WifiNode {
+ private:
+  TaskHandle_t taskHandle;
```

Comment:
to mantain consistency, we could call this taskHandle something like we did on gateway.h "annouceTaskHandle"

Reply:
Done.